doc/UserGuide.md

```diff
@@ -111,9 +124,13 @@ Examples:
   `viewall 1`<br>
   Views all details of the 1st person in the results of the `find` command.
 
+### Sort addressbook : `sort`
+Sort addressbook alphabetically.<br>
```

Comment:
What do you mean by sort addressbook? How is sorting carried out? For name, address or both?